x-pack/plugins/endpoint/public/applications/endpoint/index.tsx

```diff
@@ -53,15 +54,7 @@ const AppRoot: React.FunctionComponent<RouterProps> = React.memo(({ basename, st
             render={() => {
               // FIXME: This is temporary. Will be removed in next PR for endpoint list
               store.dispatch({ type: 'userEnteredEndpointListPage' });
```

Comment:
We should use the `usePageId` hook instead.

Reply:
ok